test/e2e/configuration_test.go

```diff
@@ -0,0 +1,144 @@
+package e2e_test
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/pborman/uuid"
+
+	. "github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+
+	cli "github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+var _ = Describe("Deploy simple cluster", func() {
+
```

Comment:
we need to remove the cluster in `AfterEach` to ensure we don't leave rubbish, see int tests